packages/codemaker/package.json

```diff
@@ -5,8 +5,8 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
+    "build": "tsc -b",
```

Comment:
What’s `-b`?

Reply:
In full, `--build`. It's the new compiler mode that supports project references.